### PR TITLE
Docker image build

### PR DESCRIPTION
**Pull request by ChinaSilence**
- fix readme
- use cache to build new image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ npm run serve
 ### Docker Build
 ```
 npm install
-npm run serve
+npm run build
```
Comment (wu-sheng) [inline README.md]: You shouldn't use `build` in my mind. Build should active auth from my understanding.

File: Dockerfile [hunk 2]
```diff
@@ -16,14 +16,15 @@
...
-COPY ./dist/ /usr/share/nginx/html/
-COPY ./nginx.conf /etc/nginx/nginx.conf
 COPY ./run.sh /usr/share/run.sh
 
 RUN set -ex \
     && apk add --no-cache bash tzdata \
     && chmod +x /usr/share/run.sh
 
+COPY ./nginx.conf /etc/nginx/nginx.conf
+COPY ./dist/ /usr/share/nginx/html/
```
Comment (wu-sheng) [inline Dockerfile]: Why do you change this?

File: README.md (hunk 1 above)
Comment (ChinaSilence) [reply README.md]: `npm run serve` just start this project for development, will not generate any resource in folder `./dist`.

Comment (wu-sheng) [reply README.md]: Get your point. @JaredTan95 consider this docker becomes the test env only, could you consider to change to as serve mode without nginx, just nodejs?

**Comment by wu-sheng** [comment]
Docker mode is not required, I think. I am going to remove it. Docker mode should only run in SkyWalking webapp container.

**Pull request closed without merging**